README.rst

```diff
@@ -1,30 +1,50 @@
-Snowflake Hive SDK
-******************
+Snowflake Hive Metastore Connector
+**********************************
+
+The Snowflake Hive metastore connector provides an easy way to query Hive-managed data via Snowflake. Once installed, the connector listens to Hive metastore events and creates the equivalent Snowflake objects.
```

Comment:
I'll add a link to docs on snowflake.net once it's available. It'll have more detailed information about the configurations.